dev/update-authors.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+// original comes from https://github.com/nodejs/node/blob/c18ca140a12b543a3f970ef379f384ebd3297c3d/tools/update-authors.js
+
+// Usage: dev/update-author.js [--dry]
+// Passing --dry will redirect output to stdout rather than write to 'AUTHORS'.
+'use strict';
+
+const { spawn } = require('child_process');
+const fs = require('fs');
+const readline = require('readline');
+
+const log = spawn(
+  'git',
+  // Inspect author name/email and body.
+  ['log', '--reverse', '--format=Author: %aN <%aE>\n%b'],
+  {
+    stdio: ['inherit', 'pipe', 'inherit'],
+  },
+);
+const rl = readline.createInterface({ input: log.stdout });
+
+let output;
+if (process.argv.includes('--dry')) {
+  output = process.stdout;
+} else {
+  output = fs.createWriteStream('AUTHORS');
+}
+
+output.write('# Authors ordered by first contribution.\n\n');
+
+const seen = new Set();
+
+// exclude emails from <ROOT>/AUTHORS file
+const excludeEmails = new Set(['<bot@renovateapp.com>', '<support@greenkeeper.io>']);
```

Comment:
With the dependabot security update we had yesterday you need to exclude some additional emails;
<support@github.com> and <49699333+dependabot[bot]@users.noreply.github.com>
But maybe it would be better to generate the AUTHORS file based on the contributors' GitHub accounts ( https://github.com/sequelize/sequelize/graphs/contributors ) which I've seen in some other projects as well

Reply:
@WikiRik things to note

From https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/viewing-a-projects-contributors
If you don't appear in a repository's contributors graph, it may be because:
- You aren't one of the top 100 contributors.
- Your commits haven't been merged into the default branch.
- The email address you used to author the commits isn't connected to your account on GitHub.

Reply:
adding filter is preferred than losing another contributors data if the contributors is more than 100.

Reply:
Ah, then it's not better to generate it based on that.
Did you think about a way of combining duplicate emails if they are both associated with a GitHub account?

Reply:
If I may add my 2 cents: I don't think duplication is a problem 😊🙈

Reply:
> If I may add my 2 cents: I don't think duplication is a problem 😊🙈

Since there's no prohibition against duplicate emails, even if it's better because it's like nodes showing changes to authors emails, at least we've filtered emails for bots. Maybe this is enough to merge. Any thoughts on this? @ephys @WikiRik I will change my habit to rebase

Reply:
> If I may add my 2 cents: I don't think duplication is a problem 😊🙈

Since there's no prohibition against duplicate emails, even if it's better because it's like nodes showing changes to authors emails, at least we've filtered emails for bots. Maybe this is enough to merge. Any thoughts on this? @ephys @WikiRik